docs/development/maintenance.md

```diff
@@ -13,104 +15,124 @@ modules of ILIAS.
+the **Authority to Sign of Conceptual Changes** gets assigned related bugs
+automatically by the [Issue-Tracker](https://mantis.ilias.de). S/he is responsible
+to make sure all issues receive a response within the defined time frame and are
+either fixed in a timely manner or postponed/closed with a solid explanation.
+* The people holding the **Authority to Sign of Code Changes** are responsible
+for pull requests to their component and get assigned related pull requests
+by the Technical Board according to the [Rules for Maintainers and Coordinators
+assigned to PRs](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/contributing.md#rules-for-maintainers-assigned-to-prs).
+* The person/people holding the **Authority to (De-)Assign Authorities**
+coordinate assignments of authorities with the Product Manager and the Technical
+Board, who hold a vetoing power over these decisions.
+
+# Additional Rules and Guidelines
+* Although the first decision on new features or feature removals in a unit of
+code lie with the person/people holding the **Authority to Sign of Conceptual
+Changes** the final decisions are made by them together with the Product
+Manager during the Jour Fixe meetings after an open discussion.
+* If nobody holds the **Authority to (De-)Assign Authorities** for a code unit
+it defaults to the Technical Board.
+* Final decision about getting write access to the ILIAS development system
+(GitHub) is handled by the Product Manager together with the Technical Board.
+* Authorities are listed with the name of the person holding the authority. In
+addition the company the person is working for can be listed, too.
+* If a company is listed for the last assignee of the **Authority to (De-)Assign
+Authorities** the company can propose a prioritized candidate for the
+succession.
+
+## Tracking Authority
+Authorities are tracked in `maintenance.json` files placed in the root of the
+corresponding code unit of ILIAS. The file contains the following fields:
+
+* **"Authority to Sign of Code Changes"**:
+    An array in the form [ `<username> (<userid>), <company> (<company_page>)` ]
+    pointing to valid users/companies on https://docu.ilias.de.
+* **"Authority to Sign of Conceptual Changes"**:
+    An array in the form [ `<username> (<userid>, <company> (<company_page>)` ]
+    pointing to valid users on https://docu.ilias.de.
+* **"Authority to Sign of Testcase Changes"**:
+    An array in the form [ `<username> (<userid>), <company> (<company_page>)` ]
+    pointing to valid users on https://docu.ilias.de.
+* **"Authority to (De-)Assign Authorities"**:
+    An array in the form [ `<username> (<userid>), <company> (<company_page>)` ]
+    pointing to valid users on https://docu.ilias.de.
+* **"tester"**:
+    An array in the form [ `<username> (<userid>), <company> (<company_page>)` ]
+    pointing to valid users on https://docu.ilias.de.
+* **"testcase_writer"**:
```

Comment:
Können wir den nicht einfach kippen? Nachdem mir Fabian K. und Hansjörg hier das Prinzip erklärt haben, scheint mir dieser Eintrag eigentlich nicht nötig zu sein.

Reply:
+1

Reply:
Wenn wir die Angaben für Tester und Testcase-Autoren kippen, wo pflegen wir dann diese Infos? Das führt dazu, dass wir sowohl für die Maintainerschaften als auch die Tester-Angelegenheiten parallele Dokumente aktualisieren müssen. Was spricht dagegen, die Angaben für Testcases und für Tester in der maintenance.md zu behalten?

Reply:
Die Tester wurden nicht gekippt, siehe e.g. hier. Die sind noch da. Die Testcase-Autoren sind in der Kompetenz "Authority to Sign off Testcase Changes" aufgegangen: Denn eigentlich ist die Bezeichnung, wie mir Fabian Kruse erklärt hat, ungenau: Es geht darum, die Testcases zu kuratieren, gepflegt werden sie von den SPs oder den Auftraggeber oder eigentlich von x-irgendwem. Meiner Meinung nach, würde die Info einfach gedoppelt.

Reply:
Danke für die Klärung. Dann hatte ich das falsch verstanden. In dem Fall bin ich völlig d'accord.